src/rgw/rgw_op.h

```diff
@@ -374,7 +374,9 @@ class RGWGetObj : public RGWOp {
   ceph::real_time unmod_time;
   ceph::real_time *mod_ptr;
   ceph::real_time *unmod_ptr;
+  ceph::real_time expiration_date;
```

Comment:
You can define these variables local to Response() .. no need to keep them in common struct here.